doc/source/user_guide/materials.rst

```diff
@@ -5,10 +5,16 @@ PyFluent supports defining materials using :ref:`ref_solver_tui_commands` and
 
 Using solver TUI commands
 -------------------------
-This example shows how you define materials using
-:ref:`ref_solver_tui_commands`.
+The following example shows a comparison between the TUI command and the
+python code for defining materials.
+
+TUI command
+
+.. code:: scheme
+
+    /define/materials elbow-fluid yes water-liquid no no no no 0 no 0 no 0 no 0 no 0 no 1 no no no no no
 
-The ``define.materials`` TUI enters the materials menu.
+Python command
 
 .. code:: python
```

Comment:
Seems to me in the TUI example we should at least include reading the case, copying the material and setting the cell zone boundary conditions